### ISSUE TITLE
Add documentation on Link Capturing in web apps

### ISSUE THREAD

**Issue by joycetoh8** [chromeos/chromeos.dev#220]
**Is your feature request related to a problem? Please describe.**
With the previous Declarative Link Capturing API being replaced by Link Capturing and the Launch Handler API, we should have documentation on chromeos.dev so developers are aware.

**Describe the solution you'd like**
Either a blog post under "News & Stories" or a page under "Apps & Games" -> "Adapting Web".

**Describe alternatives you've considered**
There is documentation on web.dev (https://web.dev/launch-handler/)

**Comment by Snugug** [comment]
@joycetoh8 did we ever do this?

**Comment by joycetoh8** [comment]
Yes, we have an article under news at https://chromeos.dev/en/posts/customize-pwa-window-launch
